test/acceptance/behave/README.md

```diff
@@ -259,6 +259,11 @@ Took 5m52.659s
 
 The log is stored in `logs` folder (if this folder does not exist it is created) and is called `behave.log` see `logging.ini`.
 
+The Context Broker must start with "DEBUG" level in "CB_EXTRA_OPS" field into properties.json file. Ex:
```

Comment:
I'd add in the sentence why this is a must, e.g.

"The Context Broker must start with ... into properties.json file, otherwise some test expecing that setup will fail"

(Please replace "otherwise some test expecing that setup will fail" by the actual reason if I'm wrong)